app/code/Magento/Bundle/Helper/Data.php

```diff
@@ -17,15 +19,23 @@ class Data extends \Magento\Framework\App\Helper\AbstractHelper
      */
     protected $config;
 
+    /**
+     * @var StockItemRepositoryInterface
+     */
+    protected $stockItemRepository;
```

Comment:
Please avoid creating new `private` methods and properties. Technical Direction of Magento favor composition over inheritance.